.github/workflows/publish-docs.yaml

```diff
@@ -6,6 +6,9 @@ on:
     branches:
       - main
 
+permissions:
+  contents: read
```

Comment:
since this one is publishing docs, will it need write permissions?

Reply:
Thanks for review.  This is the permission only for the github contents.  The list of operations are listed [here](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents).   It seems OK for me to put the `read` but I can change it to `write` for safe.

Reply:
This one needs `write`. https://github.com/peaceiris/actions-gh-pages.